### PR TITLE
new theme

### PR DESCRIPTION
**Pull request by ahmaxed**
- [ ] I have read [freeCodeCamp's contribution guidelines](https://github.com/freeCodeCamp/freeCodeCamp/blob/master/CONTRIBUTING.md).
- [ ] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [ ] My pull request targets the `master` branch of freeCodeCamp.
- [ ] None of my changes are plagiarized from another source without proper attribution.
- [ ] All the files I changed are in the same world language (for example: only English changes, or only Chinese changes, etc.)
- [ ] My changes do not use shortened URLs or affiliate links.

Closes #XXXXX

### CODE REVIEW

File: client/src/components/FourOhFour/index.js [hunk 1]
```diff
@@ -47,7 +47,7 @@ class NotFoundPage extends Component {
             </blockquote>
           </div>
         ) : (
-          <Spinner color='#006400' name='ball-clip-rotate-multiple' />
+          <Spinner color='#6B6D6F' name='ball-clip-rotate-multiple' />
```
Comment (raisedadead) [inline client/src/components/FourOhFour/index.js]: This has been updated to a newer style in the Loaded component. You should make the loader component generic and use it where ever needed

**Review by raisedadead — COMMENTED** [review]
I know this is WIP. 

But to save you some efforts I would prefer the UI components to be microscopic and their own package(s) that we can export for re-use elsewhere.

**Comment by ahmadabdolsaheb** [comment]
To do list:
- [ ] add the before after with to the `code` element. (see master layout.css)
- [ ] change the font of the guides to Lato to increase readability for larger articles.
- [ ] make sure the steel blue shades are equally away from one another
- [ ] align the search page with the theme( change background and font)
- [ ] adjust the search box size
- [ ] Made the display of the challenges change text color and links and etc according to theme(builders.js)
- [ ] change the placeholder text of the search to "search more than 10,000 lessons"
- [ ] adjust the navigation bar and center logo in all times
- [ ] adjust nav bar in mobile
- [ ] make all gray svg same color and find a color that has decent contrast in dark and light theme
- [ ] adjust the d3 svg in landing page to confirm to theme color
- [ ] change the companies logos from the landing page to an svg that could change color based on theme
- [ ] adjust the accepted privacy policy button and convey intention better
- [ ] search for instances of color and see if they could be replaced with color variables
- [ ] visit pages modals individually to check for inconsistencies.

**Comment by raisedadead** [comment]
Bump.

**Pull request closed without merging**